core/views/coreuser.py

```diff
@@ -311,8 +311,12 @@ def alert(self, request, *args, **kwargs):
                     email_address = user.email
                     preferences = user.email_preferences
                     if preferences and (preferences.get('environmental', None) or preferences.get('geofence', None)):
-                        local_zone = tz.gettz(user.user_timezone)
-                        message['date_time'] = message['date_time'].astimezone(local_zone)
+                        user_timezone = user.user_timezone
+                        if user_timezone:
+                            local_zone = tz.gettz(user_timezone)
+                            message['date_time'] = message['date_time'].astimezone(local_zone)
+                        else:
+                            message['date_time'] = time_tuple.strftime("%B %d, %Y, %I:%M %p")
```

Comment:
Can you add '(UTC)' as a suffix here so that it is clear that current timestamp is UTC @abhishek-kumar-piyush

Reply:
Done